### PR TITLE
Create beck.txt to fix issue #17

### PR DESCRIPTION
**Pull request by beck-dub**
@emilybehrendt I hope this meets your requirements.  Also, you're very mean for accusing me of breaking the repo.

### CODE REVIEW

File: beck.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- My name is Beck Weinhold
+- My favorite food is a grains/veggies/greens bowl.
```
Comment (emilybehrendt) [inline beck.txt]: beck this code has a lot of issues

Comment (emilybehrendt) [reply beck.txt]: please fix

**Comment by emilybehrendt** [comment]
closes #17

File: beck.txt (hunk 1 above)
Comment (beck-dub) [reply beck.txt]: Ahhh!

**Pull request merged**